modules/associate.js

```diff
@@ -33,10 +33,31 @@ exports.toggleAssociation = function(thread, hook, text) {
 	return true;
 };
 
+exports.printAssociations = function(api, event) {
+	var assoc = this.config[event.thread_id];
+	var message = '';
+	for (var a in assoc) {
+		message += a + ' → ' + assoc[a] + '\n';
```

Comment:
Yup, but I figured that it didn't matter cause facebook seems to just ignore the trailing whitespace. But I can change